digiwf-apps/packages/components/digiwf-date-input/package.json

```diff
@@ -24,7 +24,7 @@
     "jest": "^28.1.3",
     "jest-environment-jsdom": "^29.5.0",
     "ts-jest": "^28.0.8",
-    "uuid": "latest"
+    "uuid": "^10.0.0"
```

Comment:
Wenn ichs richtig gesehen habe wird das in der Komponente gar nicht genutzt ...
Ist die Frage ob mans dann einfach gleich entfernt

Reply:
Stimmt. Habe es entfernt.